draft-ietf-ccwg-bbr.md

```diff
@@ -1036,12 +1036,12 @@ new data, and when the delivery rate sample is not application-limited
   BBRCheckFullBWReached():
     if (BBR.full_bw_now or rs.is_app_limited)
```

Comment:
Good catch! Your fix makes the pseudocode match the Linux TCP BBR implementation of this. I can't remember now why the pseudocode diverged from the actual code.

To make it more succinct, and more closely match the Linux TCP code, can we instead please refactor your fix by changing:

if (BBR.full_bw_now or rs.is_app_limited)

to:

if (BBR.full_bw_now or !BBR.round_start or rs.is_app_limited)

Then the pseudocode will very closely match the code here:
https://elixir.bootlin.com/linux/v6.9/source/net/ipv4/tcp_bbr.c#L879